code-challenges.js

```diff
@@ -28,3 +64,26 @@ var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
+//declare a function that will take in an array
+const fullHouse = (argArray) => {
+    //Create an empty array that will hold the unique values from argument array
+    answerArray = []
+    //Map of the array to push single instance of each unique value to the new array
+    argArray.map(value => answerArray.include(value) ? "" : answerArray.push(value))
```

Comment:
with a map, you dont need to push to a new array since it will return a new array by default. what you might want is a filter.